module/VuFind/src/VuFind/Auth/LoginTokenManager.php

```diff
@@ -336,20 +345,74 @@ public function deleteUserLoginTokens($userId)
     public function deleteActiveToken()
     {
         $cookie = $this->getLoginTokenCookie();
-        if (!empty($cookie) && $cookie['series'] && $cookie['user_id']) {
-            $this->loginTokenTable->deleteBySeries($cookie['series'], $cookie['user_id']);
+        if (!empty($cookie) && $cookie['series']) {
+            $this->loginTokenTable->deleteBySeries($cookie['series']);
         }
         $this->cookieManager->clear($this->getCookieName());
     }
 
+    /**
+     * Create a new login token series or rotate login token in given series
+     *
+     * @param User   $user           User
+     * @param string $sessionId      Session identifier
+     * @param string $series         Login token series
+     * @param int    $expires        Token expiration timestamp
```

Comment:
Should this comment note the special-case behavior for a 0 value?

Reply:
I changed the special case to null and added a note.